README.md

```diff
@@ -91,14 +115,99 @@ For **Azure CLI**, create a `public.json` (and optionally `protected.json`) and
 
 
 
+_For PowerShell with variables_
```

Comment:
This looks way too detailed to me. Can't we just leave it to the reader as an exercise to extract reusable details into a variables and make this command shorter?